src/utils/__tests__/internalLinks-test.js

```diff
@@ -37,6 +37,100 @@ describe('isInternalLink', () => {
     [false, 'same domain, double slash', 'https://example.com//#narrow/stream/jest'],
     [false, 'same domain, triple slash', 'https://example.com///#narrow/stream/jest'],
 
+    // These examples may be odd-looking URLs, but are nevertheless valid.
+    [true, 'scheme-relative', '//example.com/#narrow/stream/jest'],
+    [true, 'path-relative, nop path', '.#narrow/stream/jest'],
```

Comment:
Interesting, I hadn't heard of "nop" before, but sure enough, I suppose it's at least as commonly used as "no-op": https://en.wikipedia.org/wiki/NOP_(code).